types/jest/index.d.ts

```diff
@@ -1095,7 +1147,7 @@ declare namespace jest {
      *  const mockMyFunction = myFunction as jest.MockedFunction<typeof myFunction>;
      *  expect(mockMyFunction.mock.calls[0][0]).toBe(42);
      */
-    type MockedFunction<T extends (...args: any[]) => any> = MockInstance<ReturnType<T>, ArgsType<T>> & T;
+    type MockedFn<T extends (...args: any[]) => any> = MockInstance<ReturnType<T>, ArgsType<T>> & T;
```

Comment:
I am changing the name of this type From `MockedFunction` to `MockedFn` because there is already type `MockedFunction` from `ts-jest` is present.

Reply:
(in the long run there should be only one of these types, but that could be a lot more work to figure out which'll be easier to do if we have both typed shipped side-by-side for a bit).

Reply:
> This name has to stay as is because anyone using it will now get a different type which'll break their builds unexpected so you need to rename the new `MockedFunction`.

sure 🙂